docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/element-tracking/index.md

```diff
@@ -0,0 +1,793 @@
+   snowplow('enableLinkClickTracking', { context: [componentGenerator] });
+   snowplow('enableFormTracking', { context: [componentGenerator] });
+
+   // componentGeneratorWithDetail will also populate element_detail entities for each component, but is not directly compatible with the above plugin APIs
+});
+```
+
+  </TabItem>
+  <TabItem value="browser" label="Browser (npm)">
+
+```javascript
+import { getComponentListGenerator } from '@snowplow/browser-plugin-element-tracking';
+import { enableLinkClickTracking } from '@snowplow/browser-plugin-link-click-tracking';
+import { enableFormTracking } from '@snowplow/browser-plugin-form-tracking';
+
+// access a context generator aware of the startElementTracking "components" configuration
+const [componentGenerator, componentGeneratorWithDetail] = getComponentListGenerator();
+
+// this will attach the component_parents entity to events generated by these plugins that show the component hierarchy
+enableLinkClickTracking({ options: { ... }, psuedoClicks: true, context: [componentGenerator] });
+enableFormTracking({ context: [componentGenerator] });
+
+// componentGeneratorWithDetail will also populate element_detail entities for each component, but is not directly compatible with the above plugin APIs
+```
+  </TabItem>
+</Tabs>
+
+```json reference title="Entity - iglu:com.snowplowanalytics.snowplow/component_parents/jsonschema/1-0-0" referenceLinkText="See schema on Github"
+https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/component_parents/jsonschema/1-0-0
+```
+
+#### Element contents
+
+The `element_content` schema is attached when you use the [`contents` setting](#element-data).
+
+There will likely be multiple instances of this entity in individual events, and the list of them are a flattened tree representation of the nested configuration provided.
+Each instance contains references to the parent `element_content` or `element` entity instance that contains it in the `parent_name` and `parent_index` properties (via `element_name` and `element_index`, respectively).
+Nested `details` configurations are also used to populate the `attributes` for each instance.
+
+```json reference title="Entity - iglu:com.snowplowanalytics.snowplow/element_content/jsonschema/1-0-0" referenceLinkText="See schema on Github"
+https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/element_content/jsonschema/1-0-0
+```
+
+#### Custom context
+
+Custom entities can be attached to the events generated by the plugin.
+
+- You can include `context` alongside `elements` when calling `startElementTracking`. You can pass an array of static entities _or_ a callback function that returns such an array. The function will receive the element that the event is relevant to, and the matching rule that defined the event should fire.
+- Individual rules may also contain specific `context` in the same format as above.
+
+## Examples
```

Comment:
These examples are super interesting and useful and I think that's what most users would like to see as they are trying to understand what this plugin offers (or want to copy-paste some code to their app). Since they are also wrapped in the dropdown boxes and don't take much space, can we move them to the beginning?